indigo/distribution.yaml

```diff
@@ -774,6 +774,17 @@ repositories:
       url: https://github.com/ros/bond_core.git
       version: master
     status: maintained
+  bowpmap_ros:
+    release:
```

Comment:
a release section requires you to have run a release using bloom. There are tutorials ehre: http://wiki.ros.org/bloom through that process bloom will open the pull request for you.

And please note that you should have a separate release repository from our default source repository.